src/app/components/canvas/canvas.component.ts

```diff
@@ -325,7 +329,8 @@ export class CanvasComponent {
       userID: this.user.id,
       boardID: this.boardID,
       fabricObject: JSON.stringify(pObject.toJSON(this.fabricUtils.serializableProperties)),
-      timestamp: new Date().getTime()
+      timestamp: new Date().getTime(),
+      bucketOnly:false
```

Comment:
Instead of adding this extra field, can we use the `fabricObject` field to distinguish between a bucket-only post vs a normal canvas post? So, if `fabricObject` is null, then it's bucket only, else it's also on the canvas?

This would also require some logic in `canvas.component.ts` to be updated since now all posts don't have a proper `fabricObject` field (would be null for some).